rskj-core/src/main/java/co/rsk/rpc/modules/eth/EthModule.java

```diff
@@ -357,4 +347,22 @@ private ProgramResult callConstantWithState(CallArguments args, Block executionB
                 hexArgs.getFromAddress()
         );
     }
+
+    private void handleTransactionRevert(ProgramResult res) {
```

Comment:
nitpic:

Renaming to something that doesn't mean that it will always be handled. Something like `handleTransactionRevertIfHappens`.

Reply:
thanks for the feedback! renamed the method.